.gitignore

```diff
@@ -20,3 +20,4 @@ pip-wheel-metadata
 .vscode
 Pipfile
 Pipfile.lock
+*,cover
```

Comment:
Do these show up anywhere, or should they be anchored to the base directory as `/*,cover`??

Reply:
They show up where the `.py` is when you ask for them using `annotate` command.

Reply:
Interesting; I always just pull up the annotated html version in the browser, but that could be convenient. I'll have to try it.

Reply:
Ah; OK, that's nice to know, but I like the html version better. (Probably just the color, though!)

![image](https://user-images.githubusercontent.com/61761529/111635859-52607d00-87ce-11eb-8f62-986a1daab15d.png)

vs 

![image](https://user-images.githubusercontent.com/61761529/111636160-981d4580-87ce-11eb-9748-f8c3bb774c15.png)